### DISCUSSION TITLE
Running a Remix Run project with node-windows

### DISCUSSION THREAD

**Discussion by juandelacruz-calvo** [coreybutler/node-windows#356] [Q&A]
Hello, first of all, great job, the project has saved me a few times already.

I came with a problem that has been scratching my head for weeks now. I have a server based in Windows and since the latest release of Remix Run, when I run `npm run build`, it no longer generates a ready-to-use `.js` that I can use with node-windows. 

I was wondering if you had any examples you could share in which you use node-windows by with a script path like `npm run start` which seems to be the only thing that starts Remix Run.

I'm now to Node, so this might be a newbie question, but I haven't been able to make it work so far.

Thanks!

**Comment by coreybutler** [comment]
I don't use Remix, so I don't have any specific examples.

From what I recall, Remix basically attempts to take over the backend entirely. I don't know if that includes pre-/prost processing. It sounds like Remix is assuming it will be in control of the Node process when you want node-windows to wrap it. You may be able to look at the `start` option in the `package.json` file (scrips section) to determine which file Remix is running. If that command results in the "built" file, you could use it with node-windows. If not, you'll need to talk with someone who has more experience with Remix. 

I find it a little odd that a project would just drop `npm run build`, since that is what most people are used to using for production builds. It may be worth asking the Remix folks what changed (and why) with `npm run build`.
